scripts/bump-peer-deps.js

```diff
@@ -41,7 +41,7 @@ dirs.forEach((dir) => {
       // peerDependencies to `5.4.x` we support any patch versions of `5.4` but will prevent the user from forgetting to
       // bump `@rjsf/utils` to 5.4
       //
-      peerDeps[key] = `${majorMinorMatch[1]}.x`;
+      peerDeps[key] = `>=${majorMinorMatch[1]}.x`;
```

Comment:
Package manager would no longer show the warning described in the comment right above this. Why is this changing? Is there a good reason to support e.g. using @rjsf/core v5 with @rjsf/utils v6?